docs/how-to/customise-pdf.rst

```diff
@@ -0,0 +1,260 @@
+* **LaTeX configuration**: the LaTeX packages, styling, and configuration for the PDF output, set through the `latex_elements <https://www.sphinx-doc.org/en/master/latex.html#the-latex-elements-configuration-setting>`_ dictionary in the project :file:`conf.py`. In the starter-pack, a default set of LaTeX elements is provided by the ``canonical-sphinx`` extension. Changing the LaTeX configuration requires overriding the default values loaded from the extension.
+
+This guide covers common practices and tips for customising PDF output from your documentation project using the starter pack and the ``canonical-sphinx`` extension.
+
+For basic instructions about building the PDF, see :doc:`build`.
+
+Configure document properties
+-----------------------------
+
+The `latex_documents <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_documents>`_ variable in the Sphinx :file:`conf.py` file controls properties of the intermediate LaTeX file and the final PDF output. The values are defined in a tuple of five elements:
+
+.. code-block:: python
+
+   latex_documents = [
+       (
+           'startdocname',   # Root document (e.g., 'index' or 'pdf-index')
+           'targetname.tex', # Output LaTeX file name (no spaces)
+           'title',          # Document title (can be empty to use the root doc's title)
+           'author',         # Author name(s).
+           'theme',          # Document type: 'manual' or 'howto'
+           True,             # toctree_only: if True, only include docs in toctree
+       ),
+   ]
+
+* ``startdocname``: The root document for the PDF (without the .rst extension).
+* ``targetname``: The filename for the generated LaTeX source file. The PDF file name is derived from this filename with the ``.pdf`` extension. White spaces are not allowed.
+* ``title``: The title for the PDF document on the cover page.
+* ``author``: The author(s) of the document. Use ``\\and`` to separate multiple authors.
+* ``theme``: Either ``manual`` or ``howto``.
+
+  * ``manual``: This is the default and is intended for comprehensive, book-style documentation. It produces a PDF with chapters, sections, and a table of contents. Use this for user guides, reference manuals, or any documentation that should be structured as a book.
+  * ``howto``: This type is for shorter, task-oriented documents. It produces a simpler PDF without chapters, and is best for single-topic guides or tutorials.
+
+* ``toctree_only``: Boolean. If set to True, the ``startdocname`` document itself is not included in the output, only the documents referenced by the ``toctree`` directive are included. This is useful for creating a PDF-specific index file.
+
+For more details, see `latex_documents`_ in the Sphinx documentation.
+
+
+Change PDF document filename
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the PDF output filename is derived from the ``project`` name in the :file:`conf.py` file: lowercase characters with whitespaces removed. 
```

Comment:
Could we change "white space" to something more like "blank space" or "empty space"? I don't know if we have a standard replacement for whitespace but it should be avoided where possible

Reply:
Good point! I don't think this term is generally considered non-inclusive as "whitelist" is, so it may not be in the language check list. But "blank/empty space" is a better option